### PR TITLE
feat(responsive-background): Make the website's background image responsive

### PR DESCRIPTION
**Pull request by WNemencha**
Closes #4.

### CODE REVIEW

File: src/styles/scss/index.scss [hunk 1]
```diff
@@ -1,4 +1,8 @@
-@import 'base/variables';
-@import 'base/elements';
+@import "base/variables";
```
Comment (crysfel) [inline src/styles/scss/index.scss]: I think we need to decide whether we go with double/single quotes, for consistency.

Comment (codinronan) [reply src/styles/scss/index.scss]: I (and I think most - airbnb rule) use single for JS, double for JSX, as this retains consistency with HTML.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
-            </a>
-          </button>
-        </header>
+        <section className="App__hero">
+          <header className="App__header">
+            <h1 className="App__title">Coding Coach</h1>
+            <h4 className="App__tagline">
+              Connecting developers with mentors worldwide.
+            </h4>
+            <button className="button button__white--fill">
```
Comment (AngeloGulina) [inline src/App.js]: not a big deal, but probably should have a different name for this `button__white--fill`, if you're following BEM (as it seems). Both `white` and `fill` seem modifiers. What do you think?

File: src/App.js [hunk 3]
```diff
@@ -1,27 +1,26 @@
...
-            <a 
-              href="https://docs.google.com/forms/d/1TraoygcJay39cLiBQM-ROlToy7xIgcdx_VvCQuE1lJw/edit#responses"
-              target="blank"
-              className="button__link"
-            >
-              Get Started
-            </a>
-          </button>
-        </header>
+        <section className="App__hero">
```
Comment (AngeloGulina) [inline src/App.js]: What do you think if we switch to lowercase early on?

File: src/styles/scss/index.scss (hunk 1 above)
Comment (AngeloGulina) [inline src/styles/scss/index.scss]: How do you gals/guys approach imports?
I always make a case to import things such `variables` on every file, where needed instead the "main" file.
What do you think?

File: src/styles/css/index.css [hunk 4]
```diff
@@ -38,15 +38,14 @@ body, html {
 .App {
```
Comment (AngeloGulina) [inline src/styles/css/index.css]: Generated file should go probably in a `dist` dir and being ignored. What do you think?

File: src/App.js (hunk 2 above)
Comment (reeversedev) [reply src/App.js]: Not really sure but why are we using class names with double underscores?

File: src/App.js (hunk 3 above)
Comment (emmawedekind) [reply src/App.js]: For components? They should be uppercase.

File: src/App.js (hunk 2 above)
Comment (emmawedekind) [reply src/App.js]: I agree white & fill are both modifiers.

**Comment by emmawedekind** [comment]
You have to remove `padding: 24px` on `<div className="App"`.  Otherwise you get x and y scrollbars.

**Comment by SkyzohKey** [comment]
@emmawedekind I thought I had removed it lol, will do tomorrow, I'm sick for the moment, not enough motivation to open my laptop sorry 😖...

File: src/App.js (hunk 3 above)
Comment (AngeloGulina) [reply src/App.js]: You mean writing CSS classes upper case, because it's a "component"? (Just asking, never heard of this approach).

Comment (WNemencha) [reply src/App.js]: Yeah. When you defines styles for a component, capitalize the definition so it is consistent with how its written in JSX :p

File: src/styles/css/index.css (hunk 4 above)
Comment (WNemencha) [reply src/styles/css/index.css]: It already is ignored. I just removed the two files as they are not needed anymore.

**Comment by emmawedekind** [comment]
Awesome, could you just fix this incompatibility with es-lint please? :) 
image

**Comment by WNemencha** [comment]
@emmawedekind How to reproduce the error ? I did an `yarn && yarn start` and it works (except for the `Configuration for rule "react/jsx-no-bind" is invalid` but this is out of scope for this PR)

**Comment by emmawedekind** [comment]
I see issues in the terminal still, I've messaged you on the help channel. If we figure this out, I'll merge it.

**Pull request merged**